src/components/Onboarding/OnboardingModal.js

```diff
@@ -34,6 +34,7 @@ const OnboardingModal = React.memo(function OnboardingModal({
         const compactMode = width < 500 || height < 400
         return (
           <Modal
+            onClose={onComplete}
```

Comment:
Ah note that the intention here is for the Modal to be not closable see https://github.com/aragon/court-dashboard/issues/349#issuecomment-638025200. We should actually remove the cross icon :D

Reply:
Ah, cool! Will remove it then, makes sense due to how the `onComplete` function behaves. 👍